### DISCUSSION TITLE
QInput: "outer-error" prop

### DISCUSSION THREAD

**Discussion by iliubinskii** [quasarframework/quasar#12677] [Ideas / Proposals]
Currently, if some inputs have validation and some do not have it, it results in unequal spacing which looks ugly:
https://codepen.io/ilyalyu/pen/qBVvoRJ

I suggest to add boolean "outer-error" prop.
With this prop, validated input field should not allocate additional space for error.
Instead it should use the space that already exist between inputs.

I also suggest similar field for all other fields that support validation.

**Comment by MilosPaunovic** [comment]
What you're suggesting already exists - `hide-bottom-space` prop.

**Comment by iliubinskii** [comment]
Thank you for the hint.

I tried hide-bottom-space:
https://codepen.io/ilyalyu/pen/XWzGqdO

It is close to what I mean but not exactly the same.

When I use hide-bottom-space for field 2 and error is shown, it shifts down fields 3, 4.

What I meant is that error would appear in the empty space, but would not shift down other fields.

But I hope I'll be able to achieve this with CSS.
